frontend/src/layout/navigation/SideBar/SideBar.tsx

```diff
@@ -150,6 +150,20 @@ function Pages(): JSX.Element {
                             },
                         }}
                     />
+                    <FlaggedFeature flag="notebooks">
+                        <PageButton
+                            icon={<IconNotebook />}
```

Comment:
Ah, so this is the "classic" sidebar – in here, we should still use Material Icons. The Central icon looks out of place here, because it has a different stroke width. We definitely want to use Central in the new sidebar though.
It's slightly annoying, but icons are one of these rare areas where the inconsistencies of a gradual migration stand out starkly.

One other thing is that all Central icons should live in https://github.com/PostHog/icons, and not in `icons.tsx` here, since we're sharing them between the website and app.